internal/test/wmcb/wmcb_test.go

```diff
@@ -38,39 +36,22 @@ var (
 
 // TestWMCBUnit runs the unit tests for WMCB
 func TestWMCBUnit(t *testing.T) {
-	// Transfer the binary to the windows using scp
-	defer framework.SSHClient.Close()
-	sftp, err := sftp.NewClient(framework.SSHClient)
-	require.NoError(t, err, "sftp client initialization failed")
-	defer sftp.Close()
-	f, err := os.Open(*binaryToBeTransferred)
-	require.NoError(t, err, "error opening binary file to be transferred")
-	dstFile, err := sftp.Create(framework.RemoteDir + "\\" + "wmcb_unit_test.exe")
-	require.NoError(t, err, "error opening binary file to be transferred")
-	_, err = io.Copy(dstFile, f)
-	require.NoError(t, err, "error copying binary to the Windows VM")
-
-	// Forcefully close it so that we can execute the binary later
-	dstFile.Close()
+	for _, vm := range framework.WinVMs {
```

Comment:
Do you want to run unit tests on every created VM? I think the main purpose was to test networking scenarios

Reply:
This would need to be changed by whoever picks up the networking scenario. However you would need to run the WMCB test suite anyhow as that is what setups the node.